example-apps/internal-knowledge-search/env.example

```diff
@@ -0,0 +1,16 @@
+# Make a copy of this file with the name .env and assign values to variables
+
+# Your Elaticsearch credentials
+ELASTICSEARCH_URL=http://localhost:9200
+ELASTIC_API_KEY=
+
+# If you prefer using a cloud ID
+# ELASTIC_CLOUD_ID=
+
+# If you prefer using username and password
```

Comment:
Don't you need to use username/password? You can't create API keys if you authenticated with an API key.
Or maybe we  should just have in this comment: 
```
# If you're using Document Level Security, or if you just prefer using username and password
```

Reply:
Done